frontend/packages/noobaa-storage-plugin/src/components/capacity-card/capacity-card.tsx

```diff
@@ -0,0 +1,244 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { ChartDonut, ChartThemeColor, ChartThemeVariant } from '@patternfly/react-charts';
+import {
+  DashboardCard,
```

Comment:
alphabetize imports (applies to all others too)

Reply:
Done...